inc/saiqueue.h

```diff
@@ -393,6 +393,9 @@ typedef enum _sai_queue_stat_t
     /** Get packets deleted when the credit watch dog expires for VOQ System [uint64_t] */
     SAI_QUEUE_STAT_CREDIT_WD_DELETED_PACKETS = 0x00000026,
 
+    /** Queue delay watermark in nanoseconds [uint64_t] */
+    SAI_QUEUE_STAT_DELAY_WATERMARK = 0x00000027,
```

Comment:
The name gives an intuition that watermark value has been reported with a delay. Should we consider naming it differently ? For eg: SAI_QUEUE_STAT_WATERMARK_DELAY_TIME or SAI_QUEUE_STAT_DELAYED_WATERMARK_TIME.

Reply:
How about SAI_QUEUE_STAT_DELAY_WATERMARK_NS  or SAI_QUEUE_STAT_WATERMARK_NS ?

Reply:
That works.

Reply:
@j-bos  do you plan to change this ?

Reply:
updated